pocs/tests/test_observatory.py

```diff
@@ -69,20 +87,44 @@ def test_bad_mount_driver(config):
         Observatory(simulator=simulator, config=conf, ignore_local_config=True)
 
 
-def test_bad_scheduler(config):
+def test_can_observe_no_scheduler(config):
     conf = config.copy()
-    simulator = ['all']
-    conf['scheduler']['type'] = 'foobar'
-    with pytest.raises(error.NotFound):
-        Observatory(simulator=simulator, config=conf, ignore_local_config=True)
+    cameras = create_cameras_from_config(conf)
+
+    obs = Observatory(cameras=cameras,
+                      scheduler=None,
+                      config=conf
+                      )
 
+    assert obs.can_observe is False
```

Comment:
You've probably added the cameras because otherwise this is `False`. The situation for no scheduler is different from the situation with a badly named scheduler so that needs to be resolved first.